### ISSUE TITLE
High pass filter function is old

### ISSUE THREAD

**Issue by Aysegul-Sen** [EMVA1288/emva1288#183]
Hi there, this high pass filter function applies NxN box filter, N is parametric. Usage is N=5. But this high pass filter described in EMVA1288 Release 3.1. In Release 4.0, a different method introduced for high pass filter. In the new version, there are 3 filters: 7x7 box filter, 11x11 box filter and 3x3 binomial filter. I think this function needs an update in order to compatible with Release 4.0

https://github.com/EMVA1288/emva1288/blob/0a094593aad7c4e0b1e8fb0bad08ceb51c7997bb/emva1288/process/routines.py#L623

Thanks in advance.

**Comment by fariza** [comment]
you are right, would you be willing to submit a PR with the proposed changes?
Sorry for my super late reply
